### ISSUE TITLE
How to deal with spring api response

### ISSUE THREAD

**Issue by Y2Kot** [edvin/tornadofx#968]
Hello, I'm try to deal my client work with rest response from the server:
Example of request: 
`http://localhost:8080/api/tasks/
`
Getting answer:
[https://pastebin.com/PrgGPznk](https://pastebin.com/PrgGPznk)

How should I correctly extract it?

**Comment by ruckustboom** [comment]
This question would be much better suited for a platform like [StackOverflow](https://stackoverflow.com/questions/ask?tags=tornadofx). Not only is it more visible for anyone to answer, it's also more visible to help future users who may have a similar question.

**Comment by Y2Kot** [comment]
you r right. How ever I'd like to know if it possible to use just TornadoFX (I know that I can solve it by using OkHttp)

**Comment by edvin** [comment]
Yes, it's absolutely possible to consume this payload with TornadoFX. Just implement JsonModel in your domain objects and override the `updateModel` and `toJSON` functions as per instructions in the guide :)

https://edvin.gitbooks.io/tornadofx-guide/part2/JSON%20and%20REST.html
